seatunnel-engine/seatunnel-engine-storage/checkpoint-storage-plugins/checkpoint-storage-hdfs/pom.xml

```diff
@@ -31,16 +31,11 @@
     <artifactId>checkpoint-storage-hdfs</artifactId>
     <dependencies>
         <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-aws</artifactId>
-            <version>3.0.0</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-client</artifactId>
-            <version>3.0.0</version>
+            <groupId>org.apache.seatunnel</groupId>
+            <artifactId>seatunnel-hadoop3-3.1.4-uber</artifactId>
+            <version>${project.version}</version>
             <scope>provided</scope>
+            <classifier>optional</classifier>
```

Comment:
Mybe `<optional>true</optional>` ?

Reply:
> <optional>true</optional>

I don't understand what you mean. `optional` is a classifier, not a dependency parameter item.